apiExamples/links.html

```diff
@@ -0,0 +1,17 @@
+<auro-tail tail="HA" size="xs" href="https://hawaiianairlines.com/">
```

Comment:
Same thing here. I don't think we should include non-supported sizes for this example because it was already stated and clutters up the example.